PsfLauncher/main.cpp

```diff
@@ -230,6 +244,164 @@ void GetAndLaunchMonitor(const psf::json_object& monitor, std::filesystem::path
     LaunchMonitorInBackground(packageRoot, monitorExecutable->as_string().wide(), monitorArguments->as_string().wide(), wait, asAdmin, cmdShow, dirStr);
 }
 
+void LaunchInBackgroundAsAdmin(const wchar_t executable[], const wchar_t arguments[], bool wait, int cmdShow, LPCWSTR dirStr)
+{
+    std::wstring cmd = L"\"";
+    cmd.append(executable);

```

Comment:
Nothing to change, but leaving a note.  `wstring::append` expects a null-terminated string.    The code that calls `LaunchInBackgroundAsAdmin` uses null-terminated strings.  Someday in the future, this inputs should be checked for the null-termination.